app/pb_kits/playbook/pb_nav/_nav.jsx

```diff
@@ -2,30 +2,50 @@
 
 import React from 'react'
 import { Caption } from '../'
+import { buildCss } from '../utilities/props'
 
 type NavProps = {
-  link: String,
   title: String,
   orientation?: 'vertical' | 'horizontal',
+  link: String,
   children?: React.Node,
+  className?: String | Array<String>,
+  highlight?: Boolean,
+  variant?: 'normal' | 'subtle',
+  onClick?: EventHandler,
 }
-
 const Nav = ({
   title = '',
   orientation = 'vertical',
+  link = '',
   children,
-}: NavProps) => (
-  <div className={`pb_nav_list_${orientation}`}>
-    <div className="pb_nav_list_title">
-      <a className="pb_nav_list_item_link_text">
-        <Caption
-            size="md"
-            text={`${title}`}
-        />
-      </a>
+  className,
+  highlight = true,
+  variant = 'normal',
+  onClick = () => {},
+}: NavProps) => {
+  const cardCss = buildCss('pb_nav_list', variant, orientation, className, {
+    'highlight': highlight,
+  })
+  return (
+    <div className={cardCss}>
+      <If condition={title}>
+        <div className="pb_nav_list_title">
+          <a
+              className="pb_nav_list_item_link_text"
+              href={link}
+              onClick={onClick}
```

Comment:
@thestephenmarshall  I added an onclick here since this kit didn't have one. Thoughts?